### PR TITLE
Add fig to ecosystem

### PR DESCRIPTION
**Pull request by anderseknert**
Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

**Comment by anderseknert** [comment]
CC @developer-guy :)

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -30,6 +30,20 @@ integrations:
     code:
       - https://github.com/dapr/dapr
 
+  fig:
+    title: fig
+    description: Beatiful shell autocompletion for OPA and many other commands, for Mac OS
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:35]: [typo] "Beautiful"

File: docs/website/data/integrations.yaml [hunk 2]
```diff
@@ -30,6 +30,20 @@ integrations:
...
+    software:
+      - fig
+    labels:
+      category: utilities
+      layer: shell
+    code:
+      - https://github.com/open-policy-agent/contrib/tree/main/opa_fig_autocomplete
+      - https://github.com/withfig/autocomplete/blob/master/src/opa.ts
+    inventors:
+      - fig
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:45]: "Inventor" feels a little weird here. Is it required?

**Review by srenatus — APPROVED** [review]

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (anderseknert) [reply docs/website/data/integrations.yaml]: Thanks, fixed!

File: docs/website/data/integrations.yaml (hunk 2 above)
Comment (anderseknert) [reply docs/website/data/integrations.yaml:45]: It's a pretty weird wording for _any_ integration IMHO.. I read it more like "the organization behind the project or product"

**Review by srenatus — APPROVED** [review]

**Pull request merged**